src/api/cmlibs/zinc/types/sceneid.h

```diff
@@ -57,8 +57,12 @@ enum cmzn_streaminformation_scene_io_format
 	/*!< Unspecified attribute */
 	CMZN_STREAMINFORMATION_SCENE_IO_FORMAT_THREEJS = 1,
 	/*!< Export scene into ThreeJS compatible JSON file.*/
-	CMZN_STREAMINFORMATION_SCENE_IO_FORMAT_DESCRIPTION = 2
+    CMZN_STREAMINFORMATION_SCENE_IO_FORMAT_DESCRIPTION = 2,
 	/*!< Import/export scene configurations into the scene */
+    CMZN_STREAMINFORMATION_SCENE_IO_FORMAT_STL = 3,
```

Comment:
I expect we'll add BINARY_STL at some point, perhaps qualify this with ASCII or TEXT? Also in C++.